spec/requests/error_handling_spec.rb

```diff
@@ -1,16 +1,16 @@
 # frozen_string_literal: true
 
-RSpec.describe ErrorsController, type: :controller do
+RSpec.describe ErrorsController, type: :request do
   describe 'GET #not_found' do
     it 'returns http 404' do
-      get :not_found
+      get '/errors/not_found'
```

Comment:
```suggestion
      get errors_not_found_path
```

Nitpick: I prefer Rails routing helpers. It might be easier to find and change them than hardcoded paths. 

You can type in `bundle exec rake routes` to see the entire list of routes.